src/Components/Publishing/Sections/Image.tsx

```diff
@@ -31,7 +31,10 @@ export const Image: React.SFC<ImageProps> = props => {
     width,
   } = props
   const caption = image.caption || ""
-  const src = resize(image.url, { width: 1200, quality: GLOBAL_IMAGE_QUALITY })
+  const src = resize(image.url, {
+    width: layout === "feature" ? 2000 : 1200,
```

Comment:
I took a stab at this and am guessing this is where we want to make the change, but wondering if there is anywhere else I might have missed.

Also I conditionally change the resize width for articles with the feature layout because I'm assuming this gets used by other article layouts where we may not want to increase the width. Maybe you can confirm?

Reply:
We should use the image section's layout for this instance rather than the article layout, to avoid using large images when the layout is not wide.  For the fullscreen images the `ImageCollection` component should have prop `sectionLayout: "fillwidth".`

Reply:
Additionally it would be good to resize this conditionally based on the screen size, we should use `media` to resize to 2000 only if the screen size is larger than 1200px

Reply:
So the change is still in this file, but should be conditionally based on the sectionLayout instead of the article layout, correct?

Reply:
Yep exactly